package.json

```diff
@@ -33,11 +33,17 @@
     "posttest": "xo src lib bin"
   },
   "dependencies": {
+    "@babel/runtime": "^7.4.5",
```

Comment:
Is babel runtime needed as a normy dependency? What does this do? Does it change how anything works?

Reply:
The use of `@babel/runtime` and `@babel/plugin-transform-runtime` is suppose to cut down on potential code bloat. Without `@babel/runtime`, certain injected functions might be defined repeated throughout the code thus resulting in an increased bundle size. `@babel/runtime` makes so that those functions are imported only once through the bundle.

Here is an [example](https://babeljs.io/docs/en/babel-runtime) of how `@babel/runtime` is used.

Reply:
Thanks for the clarification :+1: